ext/opcache/config.m4

```diff
@@ -248,6 +252,9 @@ int main() {
 	Optimizer/zend_dump.c,
 	shared,,-DZEND_ENABLE_STATIC_TSRMLS_CACHE=1,,yes)
 
+  PHP_CHECK_LIBRARY(rt, shm_unlink, [PHP_ADD_LIBRARY(rt,1,OPCACHE_SHARED_LIBADD)])
+
```

Comment:
May be add -lrt in previous chunk?

Reply:
Sorry ?

Reply:
-lrt is necessary only with HAVE_SHM_MMAP_POSIX.

Reply:
right, fixeed